docs/tre-admins/setup-instructions/cicd-pre-deployment-steps.md

```diff
@@ -68,7 +68,7 @@ Configure the following secrets in your github environment:
 | `ACR_NAME` | A globally unique name for the Azure Container Registry (ACR) that will be created to store deployment images. |
 | `EXTERNAL_KEY_STORE_ID` | Optional. The ID of the external Key Vault to store CMKs in. Should not be set if `ENCRYPTION_KV_NAME` is set and only required if `ENABLE_CMK_ENCRYPTION` is true. |
 | `ENCRYPTION_KV_NAME` | Optional. The name of the Key Vault for encryption keys. Should not be set if `EXTERNAL_KEY_STORE_ID` is set and only required if `ENABLE_CMK_ENCRYPTION` is true. |
-
+| `CI_CACHE_ACR_NAME` | Optional. The name of the Azure Container Registry (ACR) used for caching in CI/CD workflows. This secret is injected from parent workflows and not needed to be present in GitHub environment secrets. |
```

Comment:
This section instructs to add the following secrets, but this specific secret can't and really shouldn't be added. So why document it here?

Maybe a better approach would be the print a specific message in the workflows if the ACR is missing which will tell the user what's wrong?

Reply:
Added. Do you think still should be removed from docs? I think it is still valuable

Reply:
If it is specified, does it have any effect? We do get questions about it. If it's the same as `ACR_NAME`, why do we give it a different variable name?

Reply:
1. I don't believe this will have effect if provided as a root secret.
2. This section where Liza added it instructs the user to add this value which I don't think is right.

Reply:
I believe we agreed this change should be removed from the PR as we don't ask the user to configure the secret